project/legislation/propublica/congress/votes.py

```diff
@@ -0,0 +1,91 @@
+import datetime
+
+from .client import Client
+from .utils import CURRENT_CONGRESS, check_chamber, parse_date
+
+
+class VotesClient(Client):
+
+    # date-based queries
+    def votes_by_day(self, chamber, date=datetime.datetime.today()):
+        """
+        Votes cast in a single day, defaults to today.
+        """
+        date = parse_date(date)
+        return self.votes_by_range(chamber, date, date)
+    
+    def votes_by_month(self, chamber, year=None, month=None):
+        """
+        Return votes for a single month, defaulting to the current month.
+        """
+        check_chamber(chamber)
+
+        now = datetime.datetime.now()
+        year = year or now.year
+        month = month or now.month
+
+        path = "{chamber}/votes/{year}/{month}.json".format(
+            chamber=chamber, year=year, month=month)
+        return self.fetch(path, parse=lambda r: r['results'])
+
+    def votes_by_range(self, chamber, start, end):
+        """
+        Return votes cast in a chamber between two dates.
+        """
+        check_chamber(chamber)
+
+        start, end = parse_date(start), parse_date(end)
+        if start > end:
```

Comment:
This should probably just throw an exception when `start > end`. What is the edge case for when start is greater than end?